tests/tests.js

```diff
@@ -1,10 +1,108 @@
-const { expect } = require('chai')
-const { describe, it } = require('mocha')
+const chai = require('chai')
+const {
+  describe, it, beforeEach, afterEach,
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+
+const { expect } = chai
+chai.use(sinonChai)
+
+
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('returns a price with two decimal places not rounded when given a number with multiple decimal places', () => {
+      const formattedPrice = pricing.formatPrice(15.1566663)
+
+      expect(formattedPrice).to.equal(15.15)
+    })
+  })
+
+
+  describe('getEmployerContribution', () => {
+    it('returns the amount  from long term disability plan contributed by employer in dollars', () => {
+      const dollarAmount = 50
+      const price = pricing.getEmployerContribution(products.longTermDisability.employerContribution, dollarAmount)
+      expect(price).to.equal(10)
+    })
+  })
+
+
+  describe('calculateVolLifePricePerRole', () => {
+
+
+    it('returns the voluntary life price for a employee per their role', () => {
+      const role = 'ee'
+      const coverageLevel = [{ role: 'ee', coverage: 120000 }]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
+      expect(price).to.equal(42)
+    })
+  })
+
+
+  it('returns the voluntary life price for a employee', () => {
```

Comment:
This test is for `calculateVolLifePrice` so it should be in the describe block for `calculateVolLifePricePerRole`